tests/test_init.py

```diff
@@ -134,6 +286,11 @@ def test_all_shape(self, init, sgntr, shp, seed):
     def test_same_output_for_same_seed(self, init, sgntr, seed, tol):
         """Confirm that initialization functions return a deterministic output
         for a fixed seed."""
+
+        # exclude case of empty parameter list
+        if len(init(**sgntr).flatten()) == 0:
```

Comment:
Wait, aren't these the cases when an empty array is to be returned? Could check for that here :)